src/lists/list-item.jsx

```diff
@@ -416,7 +421,13 @@ const ListItem = React.createClass({
       style
     );
 
-    return React.createElement('div', {style: mergedDivStyles}, contentChildren);
+    return (
+      <div
+        style={mergedDivStyles}
+        className={className}>
```

Comment:
I am adding this now. I think this will also solve the issues from your other line notes as well.